data-prepper-api/src/main/java/org/opensearch/dataprepper/model/log/OpenTelemetryLog.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.model.log;
+
+import java.util.Map;
+
+/**
+ * An OpenTelemetry log event in Data Prepper which represents a single log data point.
+ *
+ * @see <a href="https://github.com/open-telemetry/opentelemetry-proto/blob/main/opentelemetry/proto/logs/v1/logs.proto">
+ *     OpenTelemetry Protobuf Logs Spec</a>
+ * @since 1.5
```

Comment:
Also, `@since 2.1` for this class and the methods therein.

Reply:
Done